kong/plugins/aws-lambda/schema.lua

```diff
@@ -1,3 +1,14 @@
+local function check_status(status)
+  if not status then
```

Comment:
I believe I got a null pointer when I didn't check this. I'll confirm when I'm back at the keyboard.